.github/workflows/macos_wheel.yaml

```diff
@@ -24,15 +24,53 @@ defaults:
     shell: bash -l -eo pipefail {0}
 
 jobs:
+
+  generate-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@macbuildwheel
+    with:
+      package-type: wheel
+      os: macos-arm64
+      test-infra-repository: pytorch/test-infra
+      test-infra-ref: main
+      with-xpu: disable
+      with-rocm: disable
+      with-cuda: disable
+      build-python-only: "disable"
+
+  build:
+    needs: generate-matrix
+    strategy:
+      fail-fast: false
+    name: Build and Upload Mac wheel
+    uses: pytorch/test-infra/.github/workflows/build_wheels_macos.yml@macbuildwheel
+    with:
+      repository: pytorch/torchcodec
+      ref: ""
```

Comment:
Does this build the wheel against nightly?

I noticed we are testing against nightly so probably have to build against nightly too?

Reply:
Yes, I think the `build_wheels_macos` job of test-infra is setup to build against nightly by default, and we can see from the logs that nightly is pickedup

Reply:
I don't know, but this code is common between Mac and Linux, and was inherited from Linux. Let's investigate that as a follow-up.